app/MMB/src/scenes/tripServices/hotelMenuItem/HotelMenuItem.js

```diff
@@ -45,6 +45,11 @@ export type HotelData = {|
   +checkout: string | null,
 |};
 
+const convertInputStringToDate = (date: string) => {
```

Comment:
I thought it could live in DateUtils. What do you think?

Reply:
And with some test?

Reply:
Yeah, sure, I will move it later 👍

Reply:
We actually have this possibility through a private function in DateUtils, I will make it public.